src/main/java/com/skiwi/githubhooksechatservice/mvc/beans/tasks/CommentsScanTask.java

```diff
@@ -31,73 +33,83 @@
 
 	private ChatBot chatBot;
 	
-    public CommentsScanTask(StackExchangeAPIBean stackAPI, ChatBot chatBot) {
+	/**
+	 * If any substring of a comment matches this pattern, it's probably interesting.<br>
+	 * No need for case-insensitivity since the comment is already lower-cased by the time it's being compared.<br>
+	 * It wouldn't do any harm, though.
+	 */
+	private Pattern interestingComment = Pattern.compile("[*`_]{0,3}code[*`_]{0,3}\\s*[*`_]{0,3}review[*`_]{0,3}");
```

Comment:
@Vogel612 Good point. Makes it rather easier to expand, too. The only problem with that is that now I have to write _several_ unreadable regexes to strip out the Markdown formatting, unless there's a method on StackExchangeComment that gets the plaintext version (in which case my pull request becomes about six characters changed) (and looking into it, it seems rather annoyingly possible)